packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/springboot/reqautheverything.md

```diff
@@ -19,3 +20,31 @@ class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapte
     }
 }
 ```
+
+
+Kotlin:
+```kotlin
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+class SecurityConfig2 : WebSecurityConfigurerAdapter() {
+
+    override fun configure(http: HttpSecurity?) {
+        http {
+            authorizeRequests {
+                authorize("/my-anon-page", permitAll)
+                authorize( anyRequest, authenticated)
+            }
+            oauth2Login {  }
+        }
+
+    }
+
+}
+
```

Comment:
You need to add the close code-block (```) on line 46.

Reply:
Hello @brianduffield-okta ,
Added the required changes.
Thanks!

Reply:
@AndriiMaliuta Thanks for your efforts on this update, Andrii. I currently have a few questions out to our Developer teams regarding this addition and hope to hear back shortly.